.expeditor/config.yml

```diff
@@ -3,11 +3,16 @@
 github:
   # The file where the MAJOR.MINOR.PATCH version is kept. The version in this file
   # is bumped automatically via the `built_in:bump_version` merge_action.
-  version_file: "VERSION"
+  # TODO: Change this to `VERSION` once we are live
+  version_file: "VERSION_FAKE"
   # The file where our CHANGELOG is kept. This file is updated automatically with
   # details from the Pull Request via the `built_in:update_changelog` merge_action.
-  changelog_file: "CHANGELOG.md"
+  changelog_file: "CHANGELOG_FAKE.md"
```

Comment:
Same for the changelog

Reply:
Unfortunately we will have to manually update the changelog (or sync them at some point).

Reply:
Added a note to the release.md